src/routes/diagnostic.py

```
@@ -257,19 +324,98 @@ def start_toeic_listening_question_response():
+            audio_duration_seconds=stimulus.get("audio_duration_seconds"),
+            play_count=1,
+            first_played_at=now,
+            last_played_at=now,
+        )
+        db.session.add(playback)
+        db.session.commit()
+        return _no_store({"status": "success", "playback": playback.to_dict()}, 201)
+    except ValueError as exc:
+        db.session.rollback()
+        return jsonify({"status": "error", "message": str(exc)}), 400
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)